pages/[...slug].vue

```diff
@@ -18,6 +18,7 @@
   <EpPageMeta :doc="doc" />
   <NuxtLayout>
     <div class="px-4 print:px-8 mt-10 mb-96 print:mb-0">
+      <EpLayoutBreadcrumbs :breadcrumbs="breadcrumbs" />
```

Comment:
You will might need to add `v-if` here after specifying typescript in the component
```suggestion
      <EpLayoutBreadcrumbs v-if="breadcrumbs" :breadcrumbs="breadcrumbs" />
```